SSA/Projects/FullyHomomorphicEncryption/Basic.lean

```diff
@@ -52,8 +52,11 @@ noncomputable def f : (ZMod q)[X] := X^(2^n) + 1
 --   done
 
 def ZMod.toInt (x : ZMod q) : Int := ZMod.cast x
+
+variable [ hqgt1 : Fact (q > 1)] in
```

Comment:
why is this used as a scoped variable? we can just include the assumption directly right (before the `:`) below